### PR TITLE
gdal*: add page

### PR DESCRIPTION
**Pull request by JakobMiksch**
This PR adds the page for `gdal2tiles.py` ([link](https://gdal.org/programs/gdal2tiles))

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
`3.2.2`

### CODE REVIEW

File: pages/common/gdal2tiles.py.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# gdal2tiles
```
Comment (navarroaxel) [inline pages/common/gdal2tiles.py.md:1]: Should match command name
```suggestion
# gdal2tiles.py
```

Comment (JakobMiksch) [reply pages/common/gdal2tiles.py.md]: Done.

**Comment by JakobMiksch** [comment]
@navarroaxel thanks for your quick review. I changed the title in `gdal2tiles.py` as you suggested. I also fixed the errors discovered by the linter. Please let me know if there is more to do.

File: pages/common/gdal2tiles.py.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# gdal2tiles.py
+
+> Generate TMS or XYZ tiles for a raster dataset.
+> More information: <https://gdal.org/programs/gdal2tiles.html>.
+
+- Generate TMS tiles for the zoom levels 2-5 of a raster dataset:
+
+`gdal2tiles.py --zoom={{2-5}} {{input.tif}} {{output_folder}}`
```
Comment (navarroaxel) [inline pages/common/gdal2tiles.py.md:8]: We use `path/to/directory` as placeholder
```suggestion
`gdal2tiles.py --zoom={{2-5}} {{path/to/input.tif}} {{path/to/output_directory}}`
```

File: pages/common/gdal2tiles.py.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Generate XYZ tiles for the zoom levels 2-5 of a raster dataset:
+
+`gdal2tiles.py --zoom={{2-5}} --xyz {{input.tif}} {{output_folder}}`
```
Comment (navarroaxel) [inline pages/common/gdal2tiles.py.md:12]:
```suggestion
`gdal2tiles.py --zoom={{2-5}} --xyz {{path/to/input.tif}} {{path/to/output_directory}}`
```

File: pages/common/gdal_contour.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+> Create contour lines and polygons from a digital elevation model.
+> More information: <https://gdal.org/programs/gdal_contour.html>.
+
+- Create a vector dataset with contour lines every 100 meter and name the elevation property "ele":
+
+`gdal_contour -a {{ele}} -i {{100.0}} {{input.tif}} {{output.gpkg}}`
+
+- Create a vector dataset with polygons every 100 meter:
+
+`gdal_contour -i {{100.0}} -p {{input.tif}} {{output.gpkg}}`
```
Comment (navarroaxel) [inline pages/common/gdal_contour.md:12]:
```suggestion
`gdal_contour -i {{100.0}} -p {{path/to/input.tif}} {{path/to/output.gpkg}}`
```

File: pages/common/gdal_contour.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# gdal_contour
+
+> Create contour lines and polygons from a digital elevation model.
+> More information: <https://gdal.org/programs/gdal_contour.html>.
+
+- Create a vector dataset with contour lines every 100 meter and name the elevation property "ele":
+
+`gdal_contour -a {{ele}} -i {{100.0}} {{input.tif}} {{output.gpkg}}`
```
Comment (navarroaxel) [inline pages/common/gdal_contour.md:8]:
```suggestion
`gdal_contour -a {{ele}} -i {{100.0}} {{path/to/input.tif}} {{path/to/output.gpkg}}`
```

File: pages/common/gdal_translate.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# gdal_translate
+
+> Convert raster data between different formats.
+> More information: <https://gdal.org/programs/gdal_translate.html>.
+
+- Convert raster dataset to different format:
+
+`gdal_translate -of {{"JPEG"}} {{world.tif}} {{output.jpeg}}`
```
Comment (navarroaxel) [inline pages/common/gdal_translate.md:8]: The quotes should be outside the token syntax
```suggestion
`gdal_translate -of "{{JPEG}}" {{path/to/input.tif}} {{path/to/output.jpeg}}`
```

File: pages/common/gdal_translate.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Assign a projection to a raster dataset:
+
+`gdal_translate -a_srs {{EPSG:4326}} {{input.tif}} {{output.tif}}`
+
+- Reduce the size of a raster dataaset:
```
Comment (navarroaxel) [inline pages/common/gdal_translate.md:14]:
```suggestion
- Reduce the size of a raster dataset:
```

File: pages/common/gdal_translate.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`gdal_translate -outsize {{40%}} {{40%}} {{elevation.tif}} {{small.tif}}`
+
+- Convert a GeoTiff to a Cloud Optimzed GeoTiff:
+
+`gdal_translate {{input.tif}} {{output.tif}} -of COG -co COMPRESS=LZW`
```
Comment (navarroaxel) [inline pages/common/gdal_translate.md:20]:
```suggestion
`gdal_translate {{path/to/input.tif}} {{path/to/output.tif}} -of COG -co COMPRESS=LZW`
```

File: pages/common/gdal_translate.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+`gdal_translate -of {{"JPEG"}} {{world.tif}} {{output.jpeg}}`
+
+- Assign a projection to a raster dataset:
+
+`gdal_translate -a_srs {{EPSG:4326}} {{input.tif}} {{output.tif}}`
+
+- Reduce the size of a raster dataaset:
+
+`gdal_translate -outsize {{40%}} {{40%}} {{elevation.tif}} {{small.tif}}`
```
Comment (navarroaxel) [inline pages/common/gdal_translate.md:16]: which is the input and output here?

File: pages/common/gdaladdo.md [hunk 10]
```diff
@@ -0,0 +1,8 @@
+# gdaladdo
+
+> Build overview images of raster datasets.
+> More information: <https://gdal.org/programs/gdaladdo.html>.
+
+- Build overview images of a raster dataset with "average" resampling:
+
+`gdaladdo -r {{average}} {{input.tif}}`
```
Comment (navarroaxel) [inline pages/common/gdaladdo.md:8]:
```suggestion
`gdaladdo -r {{average}} {{path/to/file.tif}}`
```

File: pages/common/gdaldem.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
+# gdaldem
+
+> Tool to analyze and visualize digital elevation models(DEM).
```
Comment (navarroaxel) [inline pages/common/gdaldem.md:3]:
```suggestion
> Tool to analyze and visualize DEM (digital elevation models).
```

File: pages/common/gdaldem.md [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://gdal.org/programs/gdaldem.html>.
+
+- Create a hillshade from a DEM:
+
+`gdaldem hillshade {{input.tif}} {{output.tif}}`
```
Comment (navarroaxel) [inline pages/common/gdaldem.md:8]:
```suggestion
`gdaldem hillshade {{path/to/input.tif}} {{path/to/output.tif}}`
```

File: pages/common/gdalinfo.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
+# gdalinfo
+
+> List various information about a GDAL supported raster dataset..
```
Comment (navarroaxel) [inline pages/common/gdalinfo.md:3]:
```suggestion
> List various information about a GDAL supported raster dataset.
```

File: pages/common/gdalinfo.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://gdal.org/programs/gdalinfo.html#gdalinfo>.
```
Comment (navarroaxel) [inline pages/common/gdalinfo.md:4]:
```suggestion
> More information: <https://gdal.org/programs/gdalinfo.html>.
```

File: pages/common/gdalinfo.md [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+
+- List information about a raster dataset as JSON:
+
+`gdalinfo -json {{input.tif}}`
+
+- Show histogram values of a raster dataset:
+
+`gdalinfo -hist {{input.tif}}`
+
+- List information about a Web Map Service(WMS):
```
Comment (navarroaxel) [inline pages/common/gdalinfo.md:22]: Should be in mayus?
```suggestion
- List information about a web map service (WMS):
```

File: pages/common/gdalinfo.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+
+`gdalinfo WMS:{{https://services.meggsimum.de/geoserver/ows}}`
+
+- List information about a specific dataset of a Web Map Service(WMS):
```
Comment (navarroaxel) [inline pages/common/gdalinfo.md:26]:
```suggestion
- List information about a specific dataset of a web map service (WMS):
```

File: pages/common/gdalwarp.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
+# gdalwarp
+
+> Image reprojection and warping utility.
+> More information: <https://gdal.org/programs/gdalwarp.html>.
+
+- Reproject a raster dataset:
+
+`gdalwarp -t_srs {{EPSG:4326}} {{input.tif}} {{output.tif}}`
```
Comment (navarroaxel) [inline pages/common/gdalwarp.md:8]:
```suggestion
`gdalwarp -t_srs {{EPSG:4326}} {{path/to/input.tif}} {{path/to/output.tif}}`
```

File: pages/common/gdalinfo.md (hunk 15 above)
Comment (JakobMiksch) [reply pages/common/gdalinfo.md:22]: Web Map Service (WMS) is an established term (see https://www.ogc.org/standards/wms) and should therefore be written in capital letters

File: pages/common/gdalinfo.md (hunk 16 above)
Comment (JakobMiksch) [reply pages/common/gdalinfo.md:26]: Web Map Service (WMS) is an established term (see https://www.ogc.org/standards/wms) and should therefore be written in capital letters

**Comment by JakobMiksch** [comment]
@navarroaxel thanks for your review. I tried to apply all notes. Let me know if there is more to do.

**Review by CleanMachine1 — APPROVED** [review]
Don't know why this sat around for so long. Sorry for the wait

**Comment by JakobMiksch** [comment]
meanwhile most of the code is merged

File: pages/common/gdal2tiles.py.md [hunk 18]
```diff
@@ -0,0 +1,12 @@
+# gdal2tiles.py
+
+> Generate TMS or XYZ tiles for a raster dataset.
+> More information: <https://gdal.org/programs/gdal2tiles.html>.
```
Comment (JakobMiksch) [inline pages/common/gdal2tiles.py.md:4]:
```suggestion
> More information: <https://gdal.org/programs/gdal2tiles>.
```

**Comment by JakobMiksch** [comment]
all of the content has been merged by other PRs that's why it can be safely closed

**Pull request closed without merging**